humanapi/connect-client
# COR-3537: Support preseed mode

## What did you implement
- Support new mode called `preseed`, which is basically non-portal select mode
- Use URL interface in order to build URLs
- Add a corner case where token is not required (will need more development)
- Bump to 2.2.0

## How can we verify it
- Include https://cdn.humanapi.co/humanapi-connect-client@2.2.0-beta.1.js in your code
- In your connect integration add a button with the following attributes:
  ```html
  <button data-hapi-token="<your session token>" ​data-hapi-mode="select" ​data-hapi-preseed="true"​>Select providers</button>
  ```
- Open connect and verify that the `iframe` now is passing `...&mode=select&preseed=true` in the `src`
- Make sure ie11 doesn't break 🙄
